Plugin/BlankCanvas/README.md

```diff
@@ -0,0 +1,121 @@
+# BlankCanvas - the plugin
+
+This plugin is a blank canvas for your plugin.
+
+It offers:
+
+1. A framework for an empty plugin
+2. A sample of how to use configuration based on a default config file stored in the Plugin jar file.
+3. A checkstyle build task (to check your Java code style)
+4. 
+## How to build BlankCanvas
+
+
+1. `git clone https://github.com/sibomots/BlankCanvas.git`
+2. `cd BlankCanvas`
+3. `gradlew setupDecompWorkspace --refresh-dependencies`
+4. `gradlew build`
+
+
+## Supar Fast Way to Test (Without an IDE)
+
+1. Setup a Forge Server
```

Comment:
I agree. I forget that I strictly deal with Forge Servers and I forgot about the whole Vanilla audience.